src/apps/uclient/CMakeLists.txt

```diff
@@ -1,3 +1,37 @@
+##
```

Comment:
Why are we randomly adding license here?

Reply:
Github terms of use say that if not otherwise indicated, the license for contributions is the default license of the repository.

My reading of https://github.com/coturn/coturn/blob/master/LICENSE is that it's the 3-clause BSD license.

By explicitly listing the license in each file, it helps casual contributors to understand what the license of the project is while they are working.

Going further than that, the SPDX-License-Identifier tag near the top of files is needed for various automated software-bill-of-materials tools to properly understand the provenance of the code.